okcupyd/helpers.py

```diff
@@ -133,10 +133,15 @@ def parse_slashed_date(date_updated_text):
 
 def parse_abbreviated_date(date_updated_text):
     try:
-        parsed_time = datetime.strptime(date_updated_text, '%b %d')
+        if ', 20' in date_updated_text:
+            parsed_time = datetime.strptime(date_updated_text.replace(',',''), '%b %d %Y')
+        else:
+            parsed_time = datetime.strptime(date_updated_text, '%b %d')
     except:
         pass
     else:
+        if ', 20' in date_updated_text:
```

Comment:
what is the significance of this condition? This probably needs a comment. Is there not a better way that we can do this?